File: 2 - BackEnd/Vanilla/PHP/3 - lang/INFO.php
<?php

/* ##########==========########## */
/* ######===--- Info ---===###### */
/* ##########==========########## */

/* Muestra la información sobre: 
    --- las opciones de compilación y extensiones de PHP.
    --- Versión de PHP.
    --- Información del servidor y entorno (si se compiló como módulo).
    --- Entorno PHP.
    --- Versión del OS. 
    --- Rutas.
    --- Valor de las opciones de configuración locales y generales.
    --- Cabeceras HTTP y licencia de PHP. */
phpinfo();

/* Esta función imprime los créditos enumerando los desarrolladores, módulos, etc. de PHP. 
Genera los códigos HTML apropiados para insertar la información en una página. */
phpcredits();

# Devuelve una cadena que contiene la versión del analizador de PHP en ejecución o extensión.
phpversion();

File: 1 - BackEnd/Vanilla/Python/3 - lang/INFO.py

# ##########==========########## */
# ######===--- Info ---===###### */
# ##########==========########## */

import sys

'''
    Muestra la información sobre la versión de Python y la configuración del entorno.
'''

# Versión del analizador de Python.
print("Versión de Python:", sys.version)

# Versión detallada (major, minor, micro).
print("Versión de Python (detalles):", sys.version_info)

'''
    Información sobre el entorno de ejecución.
'''

# Ruta donde está instalado Python.
print("Ruta de instalación de Python:", sys.executable)

# Rutas donde Python busca módulos.
print("Rutas de búsqueda de módulos:", sys.path)